### PR TITLE
Nginx

### PR DESCRIPTION
**Pull request by longshuicy**
To make it work:
- Pull this branch on the tester VM
- Build nginx container; or just run the `docker-command-smile.sh` since in the yaml file i build the nginx container too
- Run the shell script `init-letsencrypt.sh` this will create the `.data/certbot` folder

### CODE REVIEW

**Comment by ywkim312** [comment]
It is not really a comment for review, just a question based on my curiosity. Why docker compose sh files, nginx folders and others are under rabbitmq folder?

**Comment by longshuicy** [comment]
> It is not really a comment for review, just a question based on my curiosity. Why docker compose sh files, nginx folders and others are under rabbitmq folder?

haha originally i was just writing rabbitmq algorithms... bad organization. I should clean this up at some point

**Comment by ywkim312** [comment]
I don't see docker-command-smile.sh. Do you mean docker-build-smile.sh? Or can you point out where I can find docker-command-smile.sh?

**Comment by longshuicy** [comment]
> I don't see docker-command-smile.sh. Do you mean docker-build-smile.sh? Or can you point out where I can find docker-command-smile.sh?

That's the shell script I shared with you privately (the one with all the environment variables)

**Comment by ywkim312** [comment]
> > I don't see docker-command-smile.sh. Do you mean docker-build-smile.sh? Or can you point out where I can find docker-command-smile.sh?
> 
> That's the shell script I shared with you privately (the one with all the environment variables)

Got it, I will use that one

**Review by ywkim312 — APPROVED** [review]

File: rabbitmq/nginx_wo_ssl/nginx_wo_ssl.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+upstream smm_smile_flask {
+    server web:8001;
```
Comment (longshuicy) [inline rabbitmq/nginx_wo_ssl/nginx_wo_ssl.conf:2]: do you mind i change this line to `smile-server:8001`? i was copying from anther project where `web` is that server's container name

Comment (ywkim312) [reply rabbitmq/nginx_wo_ssl/nginx_wo_ssl.conf:2]: smile-server is not the correct answer either. it should be actual server name

Comment (longshuicy) [reply rabbitmq/nginx_wo_ssl/nginx_wo_ssl.conf:2]: i should change this to `smm-tester.ncsa.illinois.edu:8001` ?

**Pull request merged**